containers/unit_tests/TestBitset.hpp

```diff
@@ -155,7 +155,7 @@ void test_bitset() {
 
   {
     unsigned ts = 100u;
-    bitset_type b1;
+    bitset_type b1(Kokkos::view_alloc("MyBitset"), 0);
```

Comment:
For such tests, shouldn't `Kokkos::Test::Tools` with `Config::EnableAllocs` be used ?

Reply:
Sure